### PR TITLE
clarify that you need to skip adding style-loader to your rules if using a template

### LINKED ISSUE #304: Modifying Stylus variables instructions give error with vuetifyjs/webpack template
I'm not sure where the issue exactly is but I'm posting it here since I'm following the instructions for "Modifying Stylus variables" at https://vuetifyjs.com/en/style/theme.

Steps to reproduce:

1. `vue init vuetifyjs/webpack myapp` (all enters, but choosing Yarn)
2. cd myapp
3. Following instructions from "Modifying Stylus variables" (it says "you can skip the next section." but I'm not sure which section I should skip, and the dependencies aren't already installed, so I followed the instructions anyway)
4. `yarn add -D stylus stylus-loader style-loader css-loader`
5. `vim build/webpack.base.conf.js`
6. add `{ test: /\.styl$/, loader: ['style-loader', 'css-loader', 'stylus-loader'] }` to the rules
7. `mkdir src/stylus`
8. `echo "@import '~vuetify/src/stylus/main'" > src/stylus/main.styl`
9. `vim src/main.js` and replace `import 'vuetify/dist/vuetify.min.css` with `import './stylus/main.styl'`
10. `yarn run dev`

Error is

```
 WAIT  Compiling...                                                                                                        3:47:43 pm

 94% asset optimization                                                            

 ERROR  Failed to compile with 1 errors                                                                                    3:47:43 pm

 error  in ./src/stylus/main.styl

Module build failed: ParseError: /tmp/t/myapp3/src/stylus/main.styl:4:201
   1| // style-loader: Adds some css to the DOM by adding a <style> tag
   2| 
   3| // load the styles
   4| var content = require("!!../../node_modules/css-loader/index.js??ref--13-1!../../node_modules/postcss-loader/lib/index.js??ref--13-2!../../node_modules/stylus-loader/index.js??ref--13-3!./main.styl");
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------^
   5| if(typeof content === 'string') content = [[module.id, content, '']];
   6| if(content.locals) module.exports = content.locals;
   7| // add the styles to the DOM

expected "indent", got ";"

    at Parser.error (/tmp/t/myapp3/node_modules/stylus/lib/parser.js:259:11)
    at Parser.expect (/tmp/t/myapp3/node_modules/stylus/lib/parser.js:287:12)
    at Parser.block (/tmp/t/myapp3/node_modules/stylus/lib/parser.js:845:12)
    at Parser.selector (/tmp/t/myapp3/node_modules/stylus/lib/parser.js:1555:24)
    at Parser.ident (/tmp/t/myapp3/node_modules/stylus/lib/parser.js:1446:25)
    at Parser.stmt (/tmp/t/myapp3/node_modules/stylus/lib/parser.js:778:26)
    at Parser.statement (/tmp/t/myapp3/node_modules/stylus/lib/parser.js:685:21)
    at Parser.parse (/tmp/t/myapp3/node_modules/stylus/lib/parser.js:236:25)
    at Renderer.render (/tmp/t/myapp3/node_modules/stylus/lib/renderer.js:80:22)
    at /tmp/t/myapp3/node_modules/stylus-loader/index.js:165:12
    at tryCatchReject (/tmp/t/myapp3/node_modules/when/lib/makePromise.js:840:30)
    at runContinuation1 (/tmp/t/myapp3/node_modules/when/lib/makePromise.js:799:4)
    at Fulfilled.when (/tmp/t/myapp3/node_modules/when/lib/makePromise.js:590:4)
    at Pending.run (/tmp/t/myapp3/node_modules/when/lib/makePromise.js:481:13)
    at Scheduler._drain (/tmp/t/myapp3/node_modules/when/lib/Scheduler.js:62:19)
    at Scheduler.drain (/tmp/t/myapp3/node_modules/when/lib/Scheduler.js:27:9)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)

 @ ./src/stylus/main.styl 2:14-325 21:1-42:3 22:19-330
 @ ./src/main.js
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server ./src/main.js
```

vue-cli 2.9.3

The same error was posted at https://github.com/vuetifyjs/docs/issues/92 but I couldn't find a solution there.

### PR DESCRIPTION
**Pull request by andrewharvey**
closes #304

### CODE REVIEW

**Comment by KaelWD** [comment]
How does this affect the other locales? You've moved the template sections around a fair bit.

**Comment by andrewharvey** [comment]
> How does this affect the other locales? You've moved the template sections around a fair bit.

Good point, those will need to be updated too in the same way as the diff shows.

Unfortunately since I've changed the wording I can't update the other locales, they'll need to be updated by native speakers.

If you have any feedback on the en changes, we address them first, if not and everything is good, then we can ask translators to update the other locales?

File: lang/en/style/Theme.js [hunk 1]
```diff
@@ -18,15 +18,17 @@ export default {
...
-  stylusText5: 'Keep in mind that the relative location of node_modules may differ in your project so adjust accordingly. It is recommended that the import be located in your main application `index.js` or `client-entry.js` file. **DO NOT** import your `main.styl` inside of a component. This _will_ cause performance issues and drastically slow down **HMR** (hot module reloading).',
-  stylusText6: 'After you have decided on your import location, if you are requiring the Vuetify stylesheet by a `<link>` tag in your index file, remove it. Restart your build processes and re-open your project. You should see all of the styles working correctly.',
+  stylusText3: 'This will install the dependencies needed to import and parse stylus files.',
+  stylusText4: 'If you are aren\'t using one of the pre-made templates available on the [Quick Start](/getting-started/quick-start) guide you will need to open your webpack config and add a rule for stylus. otherwise skip this step.',
+  stylusText5: 'For **SSR** based applications, ensure that the import is in your main `client-entry`.',
```
Comment (johnleider) [inline lang/en/style/Theme.js]: Change to 'For the **webpack-ssr** template, ensure...'

File: lang/en/style/Theme.js [hunk 2]
```diff
@@ -18,15 +18,17 @@ export default {
...
+  stylusText6: 'Create a folder called `stylus` in your src directory (or appropriate assets directory) with a file named `main.styl`. This will serve as the entry point to import and rebuild the default Vuetify styles. Once created, open the .styl file and add this entry.',
+  stylusText7: 'Keep in mind that the relative location of node_modules may differ in your project so adjust accordingly. It is recommended that the import be located in your main application `index.js` or `client-entry.js` file. **DO NOT** import your `main.styl` inside of a component. This _will_ cause performance issues and drastically slow down **HMR** (hot module reloading).',
```
Comment (johnleider) [inline lang/en/style/Theme.js]: Change to : 'or `client-entry.js` if using the **webpack-ssr** template'

**Comment by johnleider** [comment]
Closing due to inactivity

**Comment by bt-dd** [comment]
Yikes, this one really got me today as well. Luckily stumbled upon #304 after much googling. The error generated  -

`expected "indent", got ";"`

Has many other search results which made the root cause hard to track down. Could definitely use some clarity on the documentation still.

**Pull request closed without merging**